src/tls_wrap.cc

```diff
@@ -120,20 +117,18 @@ TLSWrap::~TLSWrap() {
 
 
 void TLSWrap::MakePending() {
-  write_item_queue_.MoveBack(&pending_write_items_);
+  write_callback_scheduled_ = true;
 }
```

Comment:
As well, I wouldn't object to removing `MakePending()` and inlining `write_callback_scheduled_ = true` at its two or three call sites.  It arguably obscures more than it abstracts.